distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -527,3 +527,7 @@ grant {
 
     permission org.osgi.service.cm.ConfigurationPermission "*", "configure";
 };
+
+grant codebase "file:/spatial-wfs-featuretransformer-xstream" {
```

Comment:
You can just add this bundle to the grant line above for `createClassLoader` that is already changed in this file. This permission doesn't have any actions, so the `"*"` is ignored anyways.